LeaderboardBackend/Models/Requests/UserRequests.cs

```diff
@@ -116,6 +116,12 @@ public record RecoverAccountRequest
     public required string Email { get; set; }
 }
 
+public record ChangePasswordRequest
+{
+    [Required]
```

Comment:
DataAnnotations attributes are ignored, only FluentValidation validators are applied, better remove it to avoid confusion